src/mesh/mesh.hpp

```diff
@@ -187,47 +188,55 @@ class MeshBlock {
   template <typename Function>
   inline void par_for(const std::string &name, const int &il, const int &iu,
                       const Function &function) {
-    parthenon::par_for(name, exec_space, il, iu, function);
+    // using loop_pattern_mdrange_tag instead of DEFAULT_LOOP_PATTERN for now
+    // as the other wrappers are not implemented yet for 1D loops
+    parthenon::par_for(loop_pattern_mdrange_tag, name, exec_space, il, iu, function);
```

Comment:
In principle this could be hacked into 3D loops (and even more so this would apply to other pieces, too, e.g., we could even go as far as expressing everything in terms of 4D loops).
On the other hand, I'm not sure that it'd clean up the code too much and we should measure overhead first.
How strongly do you feel about this? Should we address this now, open an issue to keep track of it, or is the TODO/warning fine for now?

Reply:
It's not a huge deal. The TODO/warning is fine for now. 

Actually maybe let's just add a comment about this in the docs too... just so somebody doesn't report incorrect numbers without looking at the code.